exec/exec.go

```diff
@@ -105,18 +108,36 @@ func Output(ctx context.Context, command ...string) ([]byte, []byte, error) {
 	}
 
 	cmd := exec.CommandContext(ctx, command[0], arg...)
-
-	var outbuf, errbuf bytes.Buffer
 	cmd.Stdout = &outbuf
 	cmd.Stderr = &errbuf
 
 	if err := cmd.Start(); err != nil {
-		return nil, nil, errors.Wrapf(err, "unable to run command %s", cmd)
+		return nil, nil, 0, errors.Wrapf(err, "unable to run command %s", cmd)
+	}
+
+	err := cmd.Wait()
```

Comment:
no, because `err` may contain valid error or exit code, the next line will check if the error is valid and return it back to a caller